### DISCUSSION TITLE
Unable to push publish plugins

### DISCUSSION THREAD

**Discussion by freddyucv** [dotCMS/core#21590] [General]
Issue: https://github.com/dotCMS/core/issues/21568

**Comment by freddyucv** [comment]
so should it be like just to add permission to a folder in docker?

**Comment by dsilvam** [comment]
@freddyucv not really. Read @wezell 's comment.
>When bundling OSGI plugins, we should use a subfolder of the normal /asset/bundle folder as that folder has the proper permissions, is shared and gets automatically cleaned up etc...

is just about using the proper folder

**Reply by freddyucv** [reply]
> @freddyucv not really. Read @wezell 's comment.
> 
> > When bundling OSGI plugins, we should use a subfolder of the normal /asset/bundle folder as that folder has the proper permissions, is shared and gets automatically cleaned up etc...
> 
> is just about using the proper folder

@dsilvam so then when we are running in docker, aren't we using the same folder?

**Comment by victoralfaro-dotcms** [comment]
@freddyucv @dsilvam does this mean that we will have two folders for OSGi bundles, one for docker instances and the other for instances out of any container? I will certainly like more to have just one, the one that Will suggests in the issue makes sense to me.
